interfaces/src/db.ts

```diff
@@ -116,6 +116,7 @@ export interface ClientContext {
 }
 
 export interface ServerOnlyContext {
+  verifiedToken: object | undefined; // Set on server from JWT
```

Comment:
This can be written in the typescript syntax `verifiedToken?: object`, with the same JSON schema, but allowing omitting the property completely in a literal I think.